RELEASE/scripts/autoscend/auto_acquire.ash

```diff
@@ -70,7 +70,10 @@ boolean canPull(item it, boolean historical)
 	{
 		return false;
 	}
-	
+	if (!auto_is_valid(it))
+	{
+		return false;
+	}
 	if(storage_amount(it) > 0)
```

Comment:
Smut Orc box was getting pulled in G-Lover because the auto_is_valid check was never reached.